mm-mock-server/src/mocks/util/generators.ts

```diff
@@ -92,6 +92,20 @@ export function generateChannelInboxItemMessage(channelId: string, sender: any)
     }
 }
 
+export function generateUserProfileCompletionPercentage() {
+    // Returns a random integer between 0 and 100
+    return  Math.floor(Math.random() * 100); 
+}
+
+
+export function generateUserLastUpdateTime() {
+    // Returns a random date time from one year before to now
+    var startDate = (new Date(new Date().setFullYear(new Date().getFullYear() - 1))).valueOf(); 
+    var endDate = (new Date()).valueOf();
+    return new Date(startDate + Math.random() * (endDate - startDate)); 
```

Comment:
Nit: I believe the faker library can help with generating this random data with less code.

Reply:
Great tip! Changed to faker library.